[2023-04-17 16:37:26,819] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: af_foods.transform manual__2023-04-17T15:36:25.226339+00:00 [queued]>
[2023-04-17 16:37:26,824] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: af_foods.transform manual__2023-04-17T15:36:25.226339+00:00 [queued]>
[2023-04-17 16:37:26,825] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-04-17 16:37:26,825] {taskinstance.py:1357} INFO - Starting attempt 2 of 2
[2023-04-17 16:37:26,825] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-04-17 16:37:26,832] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): transform> on 2023-04-17 15:36:25.226339+00:00
[2023-04-17 16:37:26,836] {standard_task_runner.py:52} INFO - Started process 468 to run task
[2023-04-17 16:37:26,840] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'af_foods', 'transform', 'manual__2023-04-17T15:36:25.226339+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/af_foods.py', '--cfg-path', '/tmp/tmpetu4g_uh', '--error-file', '/tmp/tmpmlkzro72']
[2023-04-17 16:37:26,841] {standard_task_runner.py:80} INFO - Job 8: Subtask transform
[2023-04-17 16:37:26,887] {task_command.py:369} INFO - Running <TaskInstance: af_foods.transform manual__2023-04-17T15:36:25.226339+00:00 [running]> on host b016b1d76fa3
[2023-04-17 16:37:28,438] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=me
AIRFLOW_CTX_DAG_ID=af_foods
AIRFLOW_CTX_TASK_ID=transform
AIRFLOW_CTX_EXECUTION_DATE=2023-04-17T15:36:25.226339+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-17T15:36:25.226339+00:00
[2023-04-17 16:37:56,267] {logging_mixin.py:115} INFO - Foods df created.
[2023-04-17 16:37:59,138] {local_task_job.py:156} INFO - Task exited with return code Negsignal.SIGKILL
[2023-04-17 16:37:59,250] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=af_foods, task_id=transform, execution_date=20230417T153625, start_date=20230417T163726, end_date=20230417T163759
[2023-04-17 16:37:59,408] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
